src/rules/cognitive-complexity.ts

```diff
@@ -142,7 +146,11 @@ const rule: Rule.RuleModule<string, Options> = {
         if (isFileComplexity) {
           // as issues are the only communication channel of a rule
           // we pass data as serialized json as an issue message
-          context.report({ node, message: fileComplexity.toString() });
+          context.report({
+            node,
+            messageId: 'refactorFunction',
```

Comment:
we should use another message id here, to have final message to contain only `fileComplexity` value (https://github.com/SonarSource/SonarJS/blob/master/eslint-bridge/src/linter.ts#L389)